packages/react-menu/tsconfig.json

```diff
@@ -1,23 +1,16 @@
 {
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "baseUrl": ".",
+    "target": "ES5",
+    "lib": ["ES5", "dom"],
     "outDir": "dist",
-    "target": "es5",
-    "module": "commonjs",
     "jsx": "react",
     "declaration": true,
-    "sourceMap": true,
     "experimentalDecorators": true,
     "importHelpers": true,
     "noUnusedLocals": true,
```

Comment:
These should probably be in the defaults too, or at least `noUnusedLocals` should (it's not included by default with `strict`)

Reply:
Hmm switching to enforcement with lint instead of TS seems reasonable. If we're going to make that change we should probably have an issue to track it. Though until that's implemented it might still be worth having `noUnusedLocals` in the default config.

Reply:
> If we're going to make that change we should probably have an issue to track it. 

I was thinking this should be rather an RFC from point of DX change , than an issue.